inspectors/models/notation/sectionbreaks/sectionbreaksettingsmodel.cpp

```diff
@@ -0,0 +1,62 @@
+#include "sectionbreaksettingsmodel.h"
+
+SectionBreakSettingsModel::SectionBreakSettingsModel(QObject* parent, IElementRepositoryService* repository) :
+    AbstractInspectorModel(parent, repository)
+{
+    setModelType(TYPE_SECTIONBREAK);
+    setTitle(tr("Section Break"));
+    createProperties();
+}
+
+void SectionBreakSettingsModel::createProperties()
+{
+    m_startWithLongInstrNames = buildPropertyItem(Ms::Pid::START_WITH_LONG_NAMES, [this] (const int pid, const QVariant& newValue) {
+        onPropertyValueChanged(static_cast<Ms::Pid>(pid), newValue);
+
+        emit requestReloadPropertyItems();
+    });
+
+    m_resetBarNums = buildPropertyItem(Ms::Pid::START_WITH_MEASURE_ONE, [this] (const int pid, const QVariant& newValue) {
+        onPropertyValueChanged(static_cast<Ms::Pid>(pid), newValue);
+
+        emit requestReloadPropertyItems();
+    });
+
+    m_pause = buildPropertyItem(Ms::Pid::PAUSE);
+}
+
+void SectionBreakSettingsModel::requestElements()
+{
+    m_elementList = m_repository->findElementsByType(Ms::ElementType::LAYOUT_BREAK);
+}
+
+void SectionBreakSettingsModel::loadProperties()
+{
+    loadPropertyItem(m_startWithLongInstrNames);
+    loadPropertyItem(m_resetBarNums);
+    loadPropertyItem(m_pause, [] (const QVariant& elementPropertyValue) -> QVariant {
+        return QString::number(elementPropertyValue.toDouble(), 'f', 2).toDouble();
```

Comment:
I think you need to have a separate class with formatting methods.
```
class DataFormat
{
public:
     static String formatSec(const QVariant& val); 
}
...
loadPropertyItem(m_pause, [] (const QVariant& elementPropertyValue) -> QVariant {
        return DataFormat::formatSec(elementPropertyValue);
}

Reply:
Everything related to the transition to the new framework -  needs to be done later and certainly not within the framework of this PR

Reply:
> Everything related to the transition to the new framework - needs to be done later and certainly not within the framework of this PR
> 
> this also applies to other similar comments

The `DataFormat` class can be created in any case and then moved to the right place.